systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBridgeExampleClients.java

```diff
@@ -178,6 +178,89 @@ public JobBuilder producerStrimziBridge() {
+                .withNewTemplate()
+                    .withNewMetadata()
+                        .withLabels(producerLabels)
+                    .endMetadata()
+                    .withNewSpecLike(podSpecBuilder.build())
+                        .withRestartPolicy("OnFailure")
+                        .withContainers()
+                            .addNewContainer()
+                                .withName(producerName)
+                                .withImagePullPolicy(Constants.IF_NOT_PRESENT_IMAGE_PULL_POLICY)
+                                .withImage(Environment.TEST_HTTP_PRODUCER_IMAGE)
+                                .addNewEnv()
+                                    .withName("HOSTNAME")
+                                    .withValue(bootstrapAddress)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("PORT")
+                                    .withValue(Integer.toString(port))
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("TOPIC")
+                                    .withValue(topicName)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("DELAY_MS")
+                                    .withValue(String.valueOf(delayMs))
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("MESSAGE_COUNT")
+                                    .withValue(Integer.toString(messageCount))
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName(tracingServiceEnvVarName)
+                                    .withValue(producerName)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("JAEGER_AGENT_HOST")
+                                    .withValue(jaegerAgentHost)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("OTEL_EXPORTER_JAEGER_ENDPOINT")
+                                    .withValue(jaegerCollectorUrl)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("JAEGER_SAMPLER_TYPE")
+                                    .withValue(jaegerSamplerType)
+                                .endEnv()
+                                .addNewEnv()
+                                    .withName("JAEGER_SAMPLER_PARAM")
+                                    .withValue(jaegerSamplerParam)
+                                .endEnv()
+                            .endContainer()
+                    .endSpec()
+                .endTemplate()
+            .endSpec();
+    }
+
```

Comment:
Why don't you use here the `producerStrimziBridge()` method and just add new envs?

Reply:
This are mostly optional -- most of the time you don't need tracing.
Where "if" is a pita using with this builder approach, I rather added new method.
But yeah, I can easily add it to the existing method ...